src/config.rs

```diff
@@ -151,15 +151,18 @@ impl Default for RawUpdatesConfig {
 #[derive(Debug, Serialize, Deserialize, PartialEq, Eq)]
 struct RawDirectoriesConfig {
     #[serde(default)]
-    pub custom_pages_dir: Option<PathBuf>,
+    pub custom_pages_dir: PathBuf,
 }
 
 impl Default for RawDirectoriesConfig {
     fn default() -> Self {
         Self {
             custom_pages_dir: get_app_root(AppDataType::UserData, &crate::APP_INFO)
                 .map(|path| path.join("pages"))
-                .ok(),
+                .unwrap_or_else(|_| {
+                    eprintln!("Could not locate the App Root");
+                    std::process::exit(1);
+                }),
```

Comment:
if this stays in, the unwrap should be moved up:

```rust
custom_pages_dir: get_app_root(AppDataType::UserData, &crate::APP_INFO)
    .unwrap_or_else(|_| {
        eprintln!("Could not locate the App Root");
        std::process::exit(1);
    })
    .join("pages"),
```

Reply:
Fixed